satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -329,32 +329,39 @@ func rootMutation(service *console.Service, types Types) *graphql.Object {
 				},
 			},
 			// deletes api key
-			DeleteAPIKeyMutation: &graphql.Field{
-				Type: types.APIKeyInfo(),
+			DeleteAPIKeysMutation: &graphql.Field{
+				Type: graphql.NewList(types.APIKeyInfo()),
 				Args: graphql.FieldConfigArgument{
 					FieldID: &graphql.ArgumentConfig{
-						Type: graphql.NewNonNull(graphql.String),
+						Type: graphql.NewNonNull(graphql.NewList(graphql.String)),
 					},
 				},
 				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
-					keyID, _ := p.Args[FieldID].(string)
-
-					id, err := uuid.Parse(keyID)
-					if err != nil {
-						return nil, err
-					}
-
-					key, err := service.GetAPIKeyInfo(p.Context, *id)
-					if err != nil {
-						return nil, err
+					paramKeysID, _ := p.Args[FieldID].([]interface{})
+
+					var keyIds []uuid.UUID
+					var keys []console.APIKeyInfo
+					for _, id := range paramKeysID {
```

Comment:
Should we return the error for all of the keys at the same time rather than one-by-one? Although here probably doesn't matter that much.